lib/rubocop/cop/lint/float_comparison.rb

```diff
@@ -58,6 +74,16 @@ def on_send(node)
         end
         alias on_csend on_send
 
+        def on_case(node)
+          node.when_branches.each do |when_branch|
+            when_branch.each_condition do |condition|
+              next if literal_safe?(condition)
+
+              add_offense(condition, message: MSG_CASE) if float?(condition)
```

Comment:
It looks simpler the following way.

```suggestion
              next if !float?(condition) || literal_safe?(condition)

              add_offense(condition, message: MSG_CASE)
```

Reply:
@koic 
Thank you for the suggestion!
I agree that your version is simpler and more readable. I've updated the code accordingly 🙇